### DISCUSSION TITLE
Vanish TAB + Placeholder Death per World

### DISCUSSION THREAD

**Discussion by thboehi** [Zrips/CMI#3383] [Feature Requests]
I've two suggestion

1. **/vanishedit TAB Appears**
It would be great in Vanish Edit to be able to still be visible in the Tab menu. 
It looks a bit weird when 10 person are online but only 9 visible. The players can be suspicious and stop cheating.

2.  **Death per world Placeholder**
I wanted to have a placeholder that could tell how much time you died in a particular world. I searched all over the internet, no Papi Expansion exists. It looks maybe useless for you but I've a PVP map on my server, and on the stats, deaths from this world are included. I wanted to do something like 

%cmi_equation_{cmi_player_deaths_world}+{cmi_player_deaths_minage}+{cmi_player_deaths_world_the_end}+{cmi_player_deaths_world_nether}%

I speak french, if you didn't understood something, ask me I'll try to explain again.

Thanks for reading me!

**Comment by Zrips** [comment]
I will need more eplanation for vanishedit thing. As players should not see users names who are vanished and they cant see them

**Comment by thboehi** [comment]
If an admin does /vanish, he **disappears from the TAB**, and becomes invisible.

The idea would be that when you do /vanish, you become invisible, but you don't disappear from the TAB.

**Comment by thboehi** [comment]
Thanks for adding the votetopcount. (it is missing on your website)

Have you looked for this feature request?

**Comment by CloudeLecaw** [comment]
death and kills per world does exist as placeholder api expansion
https://api.extendedclip.com/expansions/perworld/
